images/common/openwisp/settings.py

```diff
@@ -298,7 +300,7 @@
             'stream': sys.stdout,
         },
         'mail_admins': {
-            'level': os.environ['DJANGO_LOG_LEVEL'],
+            'level': 'ERROR',
```

Comment:
Why is the variable removed?

Reply:
This I was just unsure. If I set DJANGO_LOG_LEVEL to "INFO" ... would it send me email with every INFO log record?

Reply:
You can change this from the env, no need to hard code it:
https://github.com/openwisp/docker-openwisp/blob/master/.env#L47

Reply:
It won't send you any email unless you define the admins email in the settings, consult the django documentation for more info: https://docs.djangoproject.com/en/4.2/ref/settings/#admins

Reply:
In this case I think the best is to add another env variable for this specific line, e.g.:

```python
os.environ['DJANGO_MAIL_ADMINS_LOG_LEVEL']
```

I agree it doens't make sense to send info via email. It looks to me that this feature of sending events via email is just not used at all. I personally use sentry or other log collectors so I don't even bother with this but it would be great to fix it.

The reason for not hardcoding is that here 2 possible values make sense: ERROR and CRITICAL.